pydash/utilities.py

```diff
@@ -393,6 +394,34 @@ def noop(*args, **kargs):  # pylint: disable=unused-argument
     pass
 
 
+def nth_arg(pos):
```

Comment:
This should have a default of `0` and also be used when a non-number is given. If the `pos` is not an integer, then it should be rounded up (so always convert to int but always round up). That is how Lodash's `nthArg` works.